### PR TITLE
Generate Delete and Restore Experiment API docs

### PR DESCRIPTION
**Pull request by aarondav**

### CODE REVIEW

File: docs/source/rest-api.rst [hunk 1]
```diff
@@ -193,6 +193,89 @@ Response Structure
 
 
 
+.. _mlflowMlflowServicedeleteExperiment:
+
+Mlflow Experiments Delete
+=========================
```
Comment (stbof) [inline docs/source/rest-api.rst]: shouldn't it be Experiments Delete, not Mlflow Experiments Delete

File: docs/source/rest-api.rst [hunk 2]
```diff
@@ -193,6 +193,89 @@ Response Structure
...
+
+===========================
+
+
+
+.. _mlflowMlflowServicerestoreExperiment:
+
+Mlflow Experiments Restore
+==========================
+
```
Comment (stbof) [inline docs/source/rest-api.rst]: shouldn't it be Experiments Restore, not Mlflow Experiments Restore

File: docs/source/rest-api.rst [hunk 3]
```diff
@@ -193,6 +193,89 @@ Response Structure
...
+
+
+
+
++---------------+-----------+---------------------------------+
+|  Field Name   |   Type    |           Description           |
++===============+===========+=================================+
+| experiment_id | ``INT64`` | Identifier to get an experiment |
+|               |           | This field is required.         |
```
Comment (stbof) [inline docs/source/rest-api.rst]: Identifier to get an experiment.

File: docs/source/rest-api.rst [hunk 4]
```diff
@@ -193,6 +193,89 @@ Response Structure
...
```
Comment (stbof) [inline docs/source/rest-api.rst]: Experiment ID.

File: docs/source/rest-api.rst (hunk 3 above)
Comment (aarondav) [reply docs/source/rest-api.rst]: I changed this description to be the same as the others on this page.

**Comment by stbof** [comment]
LGTM

**Pull request merged**